### ISSUE TITLE
modify price of an existing product

### ISSUE THREAD

**Issue by Jenny-yq** [CSCI-GA-2820-SP23-001/products#5]
**As a** merchant on E-commerce 
**I need** to be able to modify the price of an existing product
**So that** the price can be appropriately adjusted due to inflation,  scarcity, or abundance
   
 ### Details and Assumptions
 * [document what you know]
   
 ### Acceptance Criteria  
   
 ```gherkin
 Given a request from a merchant to change the price of an existing product
 When the merchant put in the updated price
 Then the current price of the product should be correctly reflected while other information stays unchanged
```

**Comment by josephiswhere** [comment]
Do we want users to be able to modify the name and descriptions of their products as well?

**Comment by Jenny-yq** [comment]
Yes, they should be able to. I just wrote this story as a starting point. I will put them in separate user stories to keep each minimum. Good point though!

**Comment by Div-Mehta** [comment]
Great job on pioneering! What about currency? Do we want to add that capability? Or should that maybe be a separate story?
